### PR TITLE
Fixes #33742 - rename link to old Host Detail UI

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Word the link better

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ const ActionsBar = ({
       href={`/hosts/${hostId}`}
       key="prev-version"
     >
-      {__('Previous UI')}
+      {__('Old UI')}
```
Comment (ofedoren) [inline webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js]: Maybe we can go even further? :)
```suggestion
      {__('Legacy UI')}
```

Comment (ekohl) [reply webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js]: At this point the "legacy" is still the default. So you can also say
```suggestion
      {__('Default UI')}
```

But this naming is hard.

Comment (ezr-ondrej) [reply webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js]: As discussed on maintainers call, lets make it legacy UI.

**Review by ofedoren — APPROVED** [review]
Thanks, @ezr-ondrej :) Merging.

**Pull request merged**